dataactbroker/handlers/userHandler.py

```diff
@@ -69,6 +70,19 @@ def getUserByUID(self,uid):
         result = self.runUniqueQuery(query,"No users with that uid", "Multiple users with that uid")
         return result
 
+    def deleteUser(self, email):
+        """ Delete user with specified email.  Submissions from that user are not deleted, instead they have
+        user id set to null  """
+
+        # Find user
+        user_query = self.session.query(User).filter(User.email == email)
+        user_to_be_deleted = self.runUniqueQuery(user_query, "No user with that email", "Multiple users with that email")
+        # Set all their submissions to have no user
+        self.session.query(Submission).filter(Submission.user_id == user_to_be_deleted.user_id).update(
+            values = {"user_id":None})
```

Comment:
Good call, made this an explicit foreign key with set null and removed the now duplicated code.